sources/packages/backend/libs/test-utils/src/factories/student-assessment.ts

```diff
@@ -45,5 +48,8 @@ export function createFakeStudentAssessment(
     StudentAssessmentStatus.Submitted;
   assessment.calculationStartDate =
     options?.initialValue?.calculationStartDate ?? null;
+  assessment.previousDateChangedReportedAssessment =
+    options?.initialValue?.previousDateChangedReportedAssessment ?? null;
```

Comment:
`previousDateChangedReportedAssessment` should go into the `relations` parameters to follow what the other factories are doing.